apps/tumor/client/src/index.js

```diff
@@ -70,8 +63,10 @@ clientToConnect.onConnectionClose((httpReq) => {
 // hint: if you use the launcher.py and ws-proxy just leave out sessionURL
 // (it will be provided by the launcher)
 const config = {
-  application: 'tumor',
-  //sessionURL: 'ws://localhost:1234/ws'
+    sessionManagerURL: 'localhost:9000/paraview',
+    //sessionManagerURL: '192.168.1.18:8081/paraview',
```

Comment:
you shouldnot show your IP address on the web

Reply:
I'm removing that line though that's a local ip address